src/plugins/commit-message/index.js

```diff
@@ -30,13 +65,30 @@ const EXCLUDED_REPOSITORY_NAMES = new Set([
  */
 function checkCommitMessage(message) {
     const commitTitle = message.split(/\r?\n/)[0];
+    const errCode = [];
 
     if (message.startsWith("Revert \"")) {
-        return true;
+        return [true, []];
```

Comment:
It seems like we should change the return value to be an object so we know what `true` means. Maybe something like `return { valid: true, errors: [] }`?